src/code/entities/bank6.asm

```diff
@@ -146,20 +146,20 @@ CheckLinkInteractionWithEntity_06::
     or   [hl]                                     ; $64AE: $B6
     ld   hl, wDialogCooldown                      ; $64AF: $21 $34 $C1
     or   [hl]                                     ; $64B2: $B6
-    jr   nz, .return                              ; $64B3: $20 $0F
+    jr   nz, .skipUpdateSpeedX                              ; $64B3: $20 $0F
 
     ld   a, [wWindowY]                            ; $64B5: $FA $9A $DB
     cp   $80                                      ; $64B8: $FE $80
-    jr   nz, .return                              ; $64BA: $20 $08
+    jr   nz, .skipUpdateSpeedX                              ; $64BA: $20 $08
 
     ldh  a, [hJoypadState]                        ; $64BC: $F0 $CC
     and  J_A                                      ; $64BE: $E6 $10
-    jr   z, .return                               ; $64C0: $28 $02
+    jr   z, .skipUpdateSpeedX                               ; $64C0: $28 $02
 
     scf                                           ; $64C2: $37
     ret                                           ; $64C3: $C9
 
-.return
+.skipUpdateSpeedX
```

Comment:
Were all these changes from `.return` to `.skipUpdateSpeedX` in this file intended?

It looks like a rename scoped to a single function that accidentally leaked to all other functions…

Reply:
You're right, sorry about that. I think I've fixed all the mistaken `.skipUpdateSpeedX`s now.